norddrop/src/device/types.rs

```diff
@@ -100,6 +100,9 @@ pub enum Event {
         #[serde(flatten)]
         data: FinishEvent,
     },
+    DbLost {
+        context: String,
+    },
```

Comment:
Maybe it makes sense to wrap it into a generic `RuntimeError` type? This PR handles the database issues when opening it, but we will also encounter runtime errors with the database where we will need to recreate the database.

So we will have at least 2 database-related issues.

Reply:
Yeah, it makes sense. Let me do that